OpenBCI_32bit_Library.cpp

```diff
@@ -1676,6 +1676,76 @@ void OpenBCI_32bit_Library::csHigh(int SS)
   spi.setMode(DSPI_MODE0);  // DEFAULT TO SD MODE!
 }
 
+
+/**
+ * @description activates the LED to show marker received (500ms ON)
+ * @param `on` - [boolean] - whether to activate
+ */
+void OpenBCI_32bit_Library::activateLedMarker(boolean on){
+  ledMarkerFound = true;
+  ledState = ON;
+  nextLedEvent = millis();
+}
+
+/**
+ * @description activates the LED to SD Write (50ms:ON - 50ms:OFF )
+ * @param `on` - [boolean] - whether to activate or deactivate
+ */
+void OpenBCI_32bit_Library::activateLedSDWrite(boolean on){
+  if (on){
+    ledSDWrite = true;
+    ledState = OFF;
+  } else {
+    ledSDWrite = false;
+  }
+  nextLedEvent = millis();
+}
+
+/**
+ * @description activates the LED (either ON or OFF)
+ * @param `on` - [boolean] - whether to activate or deactivate
+ */
+void OpenBCI_32bit_Library::activateLed(boolean on){
+  if (on){
+    ledOnOff = true;
+  } else {
+    ledOnOff = false;
+  }
+  nextLedEvent = millis();
+}
+
+/**
+ * @description State machine to drive the LED based on whether:
+ 1) a marker was received
+ 2) the SD card is writing
+ 3) the LED has been signalled on or off
+ */
+void OpenBCI_32bit_Library::driveLed(void){
+  if (ledMarkerFound){
```

Comment:
Perhaps put a block to avoid this function if board is in debug aka Serial1 output mode?